fbgemm_gpu/src/split_table_batched_embeddings.cpp

```diff
@@ -115,7 +116,7 @@ TORCH_LIBRARY_FRAGMENT(fb, m) {
       "linearize_cache_indices(Tensor cache_hash_size_cumsum, Tensor indices, Tensor offsets) -> Tensor");
   DISPATCH_TO_CUDA("linearize_cache_indices", linearize_cache_indices_cuda);
   m.def(
-      "lru_cache_populate(Tensor weights, Tensor hash_size_cumsum, int total_cache_hash_size, Tensor cache_index_table_map, Tensor weights_offsets, Tensor D_offsets, Tensor linear_cache_indices, Tensor(a!) lxu_cache_state, Tensor(b!) lxu_cache_weights, int time_stamp, Tensor(c!) lru_state, bool stochastic_rounding) -> ()");
+      "lru_cache_populate(Tensor weights, Tensor hash_size_cumsum, int total_cache_hash_size, Tensor cache_index_table_map, Tensor weights_offsets, Tensor D_offsets, Tensor linear_cache_indices, Tensor(a!) lxu_cache_state, Tensor(b!) lxu_cache_weights, int time_stamp, int non_evitable_time_stamp, Tensor(c!) lru_state, bool stochastic_rounding) -> ()");
```

Comment:
Could you move `int non_evitable_time_stamp,` as the last argument and assign a default value? This is primarily for backward compatibility.

Reply:
fixed.